### ISSUE TITLE
Parsed references missing line returns

### ISSUE THREAD

**Issue by dchiu911** [rstudio/bookdown#1042]
I thought #981 might have fixed this, but the `<p>` tags are removed, so in the references section I now get

```
Reference 1
Reference 2
Reference 3
```
instead of
```
Reference 1

Reference 2

Reference 3
```

Using a custom `.csl` is a little better but there's still some alignment/line returns problems.

**Comment by cderv** [comment]
Hi, 

I don't think this is related to **bookdown** directly. I believe Pandoc has changed the default style for the references, and do not use `<p>` tags anymore. 

Try this `test.Rmd` doc with Pandoc 2.11.2 and Pandoc 2.7.3

````md
---
title: 'Style in references'
output: 
  html_document:
    keep_md: true
bibliography: packages.bib
---

Creating a bib file
```{r}
knitr::write_bib("knitr", "packages.bib")
```

We reference the package **knitr** (from @R-knitr) and its book [@knitr2015]

# References 

````

Following examples using on Pandoc, no **rmarkdown**, with this `test.md` files
```md
---
title: 'Style in reference'
references:
- author:
  - family: Xie
    given: Yihui
  id: R-knitr
  issued: 2020
  note: R package version 1.30
  title: "Knitr: A general-purpose package for dynamic report generation
    in r"
  title-short: Knitr
  type: book
  url: "https://yihui.org/knitr/"
- author:
  - family: Xie
    given: Yihui
  edition: 2nd
  id: knitr2015
  issued: 2015
  note: ISBN 978-1498716963
  publisher: Chapman; Hall/CRC
  publisher-place: Boca Raton, Florida
  title: Dynamic documents with R and knitr
  type: book
  url: "https://yihui.org/knitr/"
---

We reference the package **knitr** (from @R-knitr) and its book [@knitr2015]

# References 

```

Using Pandoc 2.11.2 and new `--citeproc`, there is no `<p>` tags.

```sh
$pandoc --version
pandoc.exe 2.11.2

$ pandoc -t html4 --citeproc test.md
<p>We reference the package <strong>knitr</strong> (from <span class="citation">Xie (2020)</span>) and its book <span class="citation">(Xie 2015)</span></p>
<h1 class="unnumbered" id="references">References</h1>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-knitr2015" class="csl-entry">
Xie, Yihui. 2015. <em>Dynamic Documents with R and Knitr</em>. 2nd
ed. Boca Raton, Florida: Chapman; Hall/CRC. <a href="https://yihui.org/knitr/">https://yihui.org/knitr/</a>.
</div>
<div id="ref-R-knitr" class="csl-entry">
ÔÇöÔÇöÔÇö. 2020. <em>Knitr: A General-Purpose Package for Dynamic Report Generation in r</em>. <a href="https://yihui.org/knitr/">https://yihui.org/knitr/</a>.
</div>
</div>
```

Using previous Pandoc 2.7.3 and `pandoc-citeproc` filter, there are some `<p>` tags
```sh
$ ~/Logiciel/pandoc/pandoc-2.7.3/pandoc --version
pandoc.exe 2.7.3

$ ~/Logiciel/pandoc/pandoc-2.7.3/pandoc -t html4 --filter pandoc-citeproc test.md
<p>We reference the package <strong>knitr</strong> (from <span class="citation">Xie</span>) and its book <span class="citation">(Xie)</span></p>
<h1 id="references" class="unnumbered">References</h1>
<div id="refs" class="references">
<div id="ref-knitr2015">
<p>Xie, Yihui. <em>Dynamic Documents with R and Knitr</em>. 2nd ed. Boca Raton, Florida: Chapman; Hall/CRC.</p>
</div>
<div id="ref-R-knitr">
<p>ÔÇöÔÇöÔÇö. <em>Knitr: A General-Purpose Package for Dynamic Report Generation in R</em>.</p>
</div>
</div>
```

So I think you'll need to use CSS to change the style - for example adding this CSS chunk could work to add the 10px margin as you had before
````md
```{css, echo = FALSE}
div#refs > div.csl-entry {
  margin-bottom: 10px
}
```
````

or using one CSL that suits you. 

However, the `gitbook` template may be missing some of the last changes in Pandoc regarding CSS for built-in `citeproc` - I'll look into it to add that.

**Comment by cderv** [comment]
I added the same CSS as in Pandoc's default html template when a CSL is used. So this should fix the issue you had when using a CSL file. (same fix as for rstudio/rmarkdown#1960)

Line spacing between entries [is defined in the CSL itself](https://docs.citationstyles.org/en/stable/specification.html#whitespace), so if you want that you need to choose a CSL accordingly or you need to edit an existing one.
